RELEASING

```diff
@@ -1,13 +1,13 @@
 Releasing an SDK Version
 ========================
 
-Create a PR which updates the changelog and bumps version
+Create a pull request which updates the changelog and bumps version
 
-Reviewer merges, then checks out `master` and runs `make release`. This will
+Reviewer merges, then checks out `main` and runs `make release`. This will
 push to pypi and tag the current commit
 
-Reviewer pushes the tag to GitHub and goes to the releases page [1] to write up
-the release, using the changelog
+Reviewer goes to the releases page [1] to write up the release, using the
+changelog
```

Comment:
This change is just because the tag should be handled automatically from now on.